x-pack/plugins/security_solution/public/cases/components/use_create_case_modal/index.tsx

```diff
@@ -22,13 +22,18 @@ export const useCreateCaseModal = ({ onCaseCreated }: Props) => {
   const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
   const closeModal = useCallback(() => setIsModalOpen(false), []);
   const openModal = useCallback(() => setIsModalOpen(true), []);
+  const onSuccess = useCallback(
+    (theCase) => {
+      onCaseCreated(theCase);
+      closeModal();
```

Comment:
nit -> to remove a depedency, you can just do that
```
const onSuccess = useCallback(
    (theCase) => {
      onCaseCreated(theCase);
      setIsModalOpen(false);
    },
    [onCaseCreated]
  );
```

Reply:
I think that it is ok to have this dependency because maybe in the future we will add more logic inside the `closeModal` function and we could forget to do it inside `onSuccess`. What do you think?

Reply:
Like I sais, it is a nit, I won't block you on that